xarray/core/types.py

```diff
@@ -19,7 +19,7 @@
         DaskArray = np.ndarray  # type: ignore
 
     try:
-        from cubed import Spec
+        from cubed import Spec  # type: ignore
```

Comment:
Does `cubed` need to go into the mypy exclusions, assuming it's not typed?

Reply:
It's not typed right now, but I would like to add some typing to it. Not sure where that means it should live right now.

Reply:
I think it should have one of these: https://github.com/pydata/xarray/blob/main/setup.cfg#L184-L185

Reply:
done

Reply:
🙏